package.json

```diff
@@ -15,9 +15,25 @@
   },
   "license": "Apache 2.0",
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
-    "eslint": "^7.13.0",
+    "@babel/eslint-parser": "^7.17.0",
+    "eslint": "^7.32.0",
     "eslint-plugin-jest": "^24.1.3",
-    "lerna": "3.22.1"
+    "lerna": "4.0.0"
+  },
+  "resolutions": {
+    "ansi-regex": "5.0.1",
```

Comment:
most of these packages are deeply nested transitive dependency that was not updated in the latest version of lerna v4.

Reply:
Do we have a ticket created to eventually remove these from our `resolutions`?